x-pack/plugins/searchprofiler/server/routes/profile.ts

```diff
@@ -56,6 +57,49 @@ export const register = ({ router, getLicenseStatus, log }: RouteDependencies) =
         log.error(err);
         const { statusCode, body: errorBody } = err;
 
+        return response.customError({
+          statusCode: statusCode || 500,
+          body: errorBody
+            ? {
+                message: errorBody.error?.reason,
+                attributes: errorBody,
+              }
+            : err,
+        });
+      }
+    }
+  );
+  router.get(
+    {
+      path: `${API_BASE_PATH}/get_Indices`,
+      validate: false,
+    },
+    async (ctx, _request, response) => {
+      const currentLicenseStatus = getLicenseStatus();
+      if (!currentLicenseStatus.valid) {
+        return response.forbidden({
+          body: {
+            message: currentLicenseStatus.message!,
+          },
+        });
+      }
+
+      try {
+        const client = (await ctx.core).elasticsearch.client.asCurrentUser;
+        const resp = await client.cat.indices({ format: 'json' });
+
+        const hasIndices = resp.length > 0;
+
+        return response.ok({
+          body: {
+            ok: true,
+            hasIndices,
```

Comment:
this is a bit strange to me, the api endpoint is called `get_indices` so I would be expecting a list of indices but in reality it returns a boolean that says whether it has indices or not.

I think we can rename this api endpoint to then be something slightly more semantic such as `has_indices` which clearly reflects that the API checks if indices exists or not. Also we dont need the `ok: true` since we have the http status code if needed.

Likewise the `useIndices` hook in the UI I think we could also rename to something like `useHasIndices`, in order to properly reflect which endpoint is calling.

Reply:
Agree, it makes more sense